multibindings.md

```diff
@@ -49,7 +50,6 @@ class Bar {
   @Inject Bar(Set<String> strings) {
```

Comment:
oh, right… this does get a little more confusing now because the changes to the providers make the assertions not make all that much sense.  What if we instead just used `TimeUnit` or something and just did assertions using well-known enum values?  Either way, we need to make sure that we update both ends so that the code clearly shows the association between the providers and what's injected.